subjects/localhost/audit/README.md

```diff
@@ -0,0 +1,67 @@
+#### Functional
+
+#### Localhost is about program your own HTTP server and test it with an actual browser.
+###### Take The necessary time to understand the project and to test it,take a look around the code to ask reasonable questions such as:
+###### Explain the basics of an HTTP server.
+###### Ask which function they used for I/O Multiplexing.
+###### Ask to get an explanation of how select (or equivalent) is working.
+###### Ask if they use only one select (or equivalent) and how they've managed the server accept and the client read/write.
```

Comment:
- Not clear from auditor perspective if the have to use only one select or not.
- Second part of the sentence ("and how ...") is not clear, please rephrase it.